NEWS.md

```diff
@@ -1,3 +1,51 @@
+# New in snapd 2.66:
+* AppArmor prompting (experimental): expand kernel support checks
+* AppArmor prompting (experimental): consolidate error messages and add error kinds
+* AppArmor prompting (experimental): grant /v2/snaps/{name} via snap-interfaces-requests-control
+* AppArmor prompting (experimental): add checks for duplicate pattern variants
+* Registry views (experimental): add handlers that commit (and cleanup) registry transactions
+* Registry views (experimental): add a snapctl fail command for rejecting registry transactions
+* Registry views (experimental): allow custodian snaps to implement registry hooks that modify and save registry data
+* Registry views (experimental): run view-changed hooks only for snaps plugging views affected by modified paths
+* Registry views (experimental): make registry transactions serialisable
+* Snap components: handle refreshing components to revisions that have been on the system before
+* Snap components: enable creating Ubuntu Core images that contain components
+* Snap components: handle refreshing components independently of snaps
+* Snap components: handle removing components when refreshing a snap that no longer defines them
+* Snap components: extend snapd Ubuntu Core installation API to allow for picking optional snaps and components to install
+* Snap components: extend kernel.yaml with "dynamic-modules", allowing kernel to define a location for kmods from component hooks
+* Desktop IDs: support installing desktop files with custom names based on desktop-file-ids desktop interface plug attr
+* Auto-install snapd on classic systems as prerequisite for any non-essential snap install
+* Support loading AppArmor profiles on WSL2 with non-default kernel and securityfs mounted
+* Debian/Fedora packaging updates
+* Add snap debug command for investigating execution aspects of the snap toolchain
+* Improve snap pack error for easier parsing
+* Add support for user services when refreshing snaps
+* Add support for store controlled forceful removal of malicious snaps 
```

Comment:
Since store part is still not available, I think we should only mention that removal with `--terminate` is now possible.

```suggestion
* Add snap remove --terminate flag for terminating running snap processes
```

Reply:
fixed